frontend/src/components/AboutMe/ProfileCard.jsx

```
@@ -17,33 +21,127 @@
+        <div className="relative group w-24 h-24 rounded-full">
+          <img
+            src={previewURL || currentPicture}
+            alt="Profile"
+            className="w-24 h-24 rounded-full object-cover border-2 border-gray-300"
+          />
+
+          {/* Hover Overlay - only shows over the image */}
+          <div
+            className="absolute inset-0 rounded-full bg-black bg-opacity-50 opacity-0 group-hover:opacity-100 flex items-center justify-center transition duration-300 cursor-pointer"
+            onClick={triggerFileInput}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)